Enable distribute application by default

justnoxx opened the pull request:
1) Enabled distribute application flag by default.
2) Updated documentation for deploy enterprise application.
3) Updated release notes.
4) Renamed syncCells to syncActiveNodes

imago-storm commented on src/main/resources/project/project.xml:
```diff
@@ -9750,9 +9750,9 @@
+                <formalParameterName>syncActiveNodes</formalParameterName>
                 <defaultValue/>
-                <description>If checked, syncCells will be triggered after deployment.</description>
+                <description>If checked, syncActiveNodes will be triggered after deployment.</description>
```
More verbose documentation required here, like "... if checked, the active nodes will be synchronized after the deployment. The flag is ignored for standalone server installation."

justnoxx commented:
Changed:
1) Enabled distribute application flag by default.
2) Updated documentation for deploy enterprise application.
3) Updated release notes.
4) Renamed syncCells property to syncActiveNodes.
